subsys/bluetooth/controller/hci_driver.c

```diff
@@ -68,6 +68,23 @@ BUILD_ASSERT(!IS_ENABLED(CONFIG_BT_CENTRAL) ||
 BUILD_ASSERT(!IS_ENABLED(CONFIG_BT_PERIPHERAL) ||
 			 (CONFIG_BT_CTLR_SDC_PERIPHERAL_COUNT > 0));
 
+#if defined(CONFIG_BT_HCI_ACL_FLOW_CONTROL)
+/*
+ * The Host will generate up to acl_pkts number of Host Number of Completed Packets command plus a
+ * number of normal HCI commands, as such we need to ensure the tx command buffer count is big
+ * enough to not block incoming ACKs from the host.
+ *
+ * When Controller to Host data flow control is supported, ensure that BT_BUF_CMD_TX_COUNT is
+ * greater than or equal to (BT_BUF_ACL_RX_COUNT + Ncmd), where Ncmd is the supported maximum
+ * Num_HCI_Command_Packets.
+ *
+ * The SDC controller (currently) does not support Num_HCI_Command_Packets > 1, which means Ncmd
+ * is always 1.
+ */
+BUILD_ASSERT(BT_BUF_ACL_RX_COUNT < CONFIG_BT_BUF_CMD_TX_COUNT,
```

Comment:
let's move it inside sdc_hci_cmd_cb_host_buffer_size_wrapper, so it all in one place

Reply:
I realized that, when building Controller-only builds `BT_BUF_ACL_RX_COUNT`  is always 0! Hence, need to use a count that is allocated in the Controller?

https://github.com/nrfconnect/sdk-zephyr/blob/9471a136b58bfa6853b4854308ca4429ba670934/include/zephyr/bluetooth/buf.h#L91-L107